test-data/snake_oil_field/snake_oil_surface.ert

```diff
@@ -41,7 +41,7 @@ GEN_DATA SNAKE_OIL_GPR_DIFF INPUT_FORMAT:ASCII RESULT_FILE:snake_oil_gpr_diff_%d
 -- surface parameters into account, they are mainly here to create a
 -- test case with surface parameters.
 
-SURFACE TOP INIT_FILES:surface/small.irap OUTPUT_FILE:surface/small_out.irap BASE_SURFACE:surface/small.irap
+SURFACE TOP INIT_FILES:surface/small.irap OUTPUT_FILE:surface/small_out.irap BASE_SURFACE:surface/small.irap FORWARD_INIT:TRUE
```

Comment:
Because there is no `%d` in the file name, meaning that ert would just load a single file and set that as the parameter value for all realizations, see more details in the other commend.